TRANSLATING.md

```diff
@@ -0,0 +1,225 @@
+# Guide for Translators
+
+This guide is written to provide instructions from scratch for contributing to the translation of the Jamulus application to other languages.
+
+There are two parts:
+
+1. Instructions for getting set up, which only needs to be done once.
+
+2. The workflow for updating and contributing translations as a part of the preparation for a release of Jamulus.
+
+----
+## Setting up
+
+The code for Jamulus is open source, and is managed and made available via the Github site.
+
+The translator must be able to do the following steps:
```

Comment:
Given the pitch of the rest of this document, I wonder if this summary is perhaps counter-productive. If somebody who didn't know _how create an account on GitHub_ saw things like "Fork the Jamulus code" and "Clone their own repository" at the start, I wouldn't blame them for hitting the back button :-)

Reply:
Yes, fair point, I’ll have a think about it.